Dockerfile.pipelines

```diff
@@ -25,11 +18,9 @@ RUN apt-get -y dist-upgrade
 # Needed for the conda and devito installs later
 RUN apt-get -y install wget bzip2 git make
 
-# Default gcc version to install
-ARG gccVersion=8
-ENV DEVITO_ARCH=gcc-$gccVersion
```

Comment:
I'm concerned by this change. This is not parametric anymore? why should it be the case?

Reply:
Not sure about these changes, was necessary for bionic.

Reply:
I think we then have to step back -- we absolutely need to test multiple gcc versions, in particular the newest ones

Reply:
We still do, it's just that now gcc-8 comes default so no need to add extra install, so it's the exact same setup as before just "easier" to install with defaults in bionic

Reply:
I think I need to regress some changes here, actually -- I was assuming (in error) something special about toolchain-r which wasn't provided by system packages.